specs/paymentrequest.html

```diff
@@ -249,6 +249,30 @@
         <a>user agent</a> while the user is providing input before approving or denying a payment request.
       </p>
 
+      <div class="issue" data-number="47" title="Should a payment request be just data, or a programmable object?">
+        Instead of a programmable object, it has been suggested that a payment
+        request should be just pure data that can be operated on by methods
+        like so:
+        <pre class="example highlight">
+// data-only payment request object example
+var request = {
+  type: 'PaymentRequest',
+  description: 'Payment to ExampleMerch for widgets',
+  acceptablePayment: {
+    paymentMethod: 'Visa',
+    paymentAmount: {
+      amount: '4.35',
+      currency: 'USD'
+    }
+  }
+};
+
+// request payment and get a promise that will resolve to
+// a payment acknowledgement
+var payment = navigator.payment.request(request);
+        </pre>
+      </div>
+
       <p>The following example shows how to construct a <a><code>PaymentRequest</code></a> and begin the
```

Comment:
@msporny Can you reword this to be a lot simpler and leave the deeper explanation for users to read in the issue itself? I would also suggest placing this issue marker alongside those for #15 and #16 which also relate to the shape of the API.

Suggestion:

```html
<div class="issue" data-number="47" title="Should a payment request be just data, or a 
programmable object?"> There is an open issue about whether the payment request should 
be a programmable object or should be just pure data that can be operated on by methods.</div>
```